clients/fossid-webapp/src/main/kotlin/ReponseBody.kt

```diff
@@ -20,14 +20,14 @@
 package org.ossreviewtoolkit.clients.fossid
 
 open class EntityPostResponseBody<T> {
-    var operation: String? = null
-    var status: Int? = null
-    var message: String? = null
-    var error: String? = null
+    val operation: String? = null
```

Comment:
While out of scope of these changes, @nnobelis probably can shed a light on this. I guess only ever one of these fields occurs in the response, and the others thus are optional.

Reply:
The data Model being not available, this is hard for me to give an accurate answer.
However from my tests, I can say that `operation` and `status` seem to be always present, `error` is only there in case of an error, `message` is there if there are no error.